public/app/plugins/panel/graph/thresholds_form.ts

```diff
@@ -30,6 +31,8 @@ export class ThresholdFormCtrl {
       fill: true,
       line: true,
       yaxis: 'left',
+      fillColor: 'rgba(234,112, 112, 0.12)',
+      lineColor: 'rgba(237, 46, 24, 0.60)',
```

Comment:
For custom color there is a color picker available that requires a color (be it hex/rgba/named color) to be passed to it via prop. Hence the fill/line color property on the threshold.

Reply:
Yes though what I mean is that in threshold_manager.ts there is code like this:
```
case 'critical': {
          fillColor = 'rgba(234, 112, 112, 0.12)';
          lineColor = 'rgba(237, 46, 24, 0.60)';
          break;
}
```
which also specifies what color there should be in case the colorMode is critical (and also for others). So I assume these two need to be in sync. Also I assume that if the colorMode change to warning the handling of the underlying color is in the threshold_manager.ts. So I wonder if it would be possible to have the `colorMode: [fillColor, lineColor]` mapping in one place or whether these are some different things that should not be deduplicated.

Reply:
yes, agree the add new threshold should no set fill & linecolor

Reply:
Ah, absolutely. Forgot to revert that change from earlier version we discussed @torkelo. Sorry about that

Reply:
Fixed in 9205b82